src/dotnet/APIView/APIViewUnitTests/LogicTest/NamespaceLogicTest/Program.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NamespaceLogicTest
+{
+    public class Program
+    {
+        public static void Main(string[] args)
```

Comment:
Can you modify the tests like other existing test classes instead of running as console program?

Reply:
Yup. don't name the test class Program as it makes .NET think it's the entry point.